from_now/from_now_test.go

```diff
@@ -0,0 +1,23 @@
+package from_now
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
```

Comment:
Use:
```go
assert "github.com/stretchr/testify/require"
```

`require` will fail the test in case of assertion violation, whereas `assert` just return a boolean to indicating if assertion failed or not.